client/src/components/Toasts/Toasts.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from 'i18n';
+
+
+const Toast = ({type, text}) => (
+  <div className={`toast toast__${type}`}>
+    <div className="toast--close">x</div>
```

Comment:
```suggestion
  <div className={`toast toast--${type}`}>
    <div className="toast__close">x</div>
```

Reply:
You've got the element and modifier around the wrong way a bunch in this PR. I'm not going to suggest changes for it all 😅

Reply:
Fixed.